src/utils/getChartOptions.ts

```diff
@@ -0,0 +1,70 @@
+import { ChartOptions } from 'chart.js';
+
+import { ProcessChartData } from '@/types';
+
+import { REGION_COLOR, LINE_COLOR } from '@/constants/chart';
+
+type Region = keyof typeof REGION_COLOR;
+
+export const getChartOptions = (
+  processChartData: ProcessChartData,
+  handleFilter: (region: string) => void,
+): ChartOptions => {
+  return {
+    interaction: {
+      mode: 'index',
+    },
+    scales: {
+      x: {
+        grid: {
+          drawOnChartArea: false,
+        },
+      },
+      bar: {
+        type: 'linear',
+        display: true,
+        position: 'left',
+        min: 10000,
+        max: 20000,
+      },
+      line: {
+        display: true,
+        position: 'right',
+        min: 0,
+        max: 100,
+      },
+    },
+    onClick: (event, activeElements) => {
+      if (activeElements.length > 0) {
+        return handleFilter(processChartData.ids[activeElements[0].index]);
+      }
+      return handleFilter('전체');
+    },
```

Comment:
handleFilter 네이밍으로 봐서는 return 없이 실행되어야 하는 것 같은데 어떤걸 리턴하는지 네이밍을 봐서는 잘 모르겠습니다

Reply:
아 handle 메서드 자체는 void 함수라 리턴해주는 값은 없습니다!

early return 을 사용해서 if문 조건에 맞으면 안에서 종료되는 용도로 썻습니다.

창환님 피드백 보니까, 아래 처럼 작성하는게 더 좋을 것 같다고 생각이 드네요!

```suggestion
      if (activeElements.length > 0) {
        handleFilter(processChartData.ids[activeElements[0].index]);
        return;
      }
      handleFilter('전체');
    },
```